src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -40,6 +42,10 @@ final class DateTimeNormalizer implements NormalizerInterface, DenormalizerInter
     public function __construct(array $defaultContext = [])
     {
         $this->setDefaultContext($defaultContext);
+
+        if (!isset($this->defaultContext[self::TIMESTAMP_CAST_KEY])) {
```

Comment:
I won't have put that in the constructor but rather in the `normalize` method, to avoid the trigger occurring too often.

Plus, we will have to create a PR to update https://github.com/symfony/recipes so that the new projects won't have any deprecations.

Reply:
Ah,okay

Reply:
updated and created recipe in https://github.com/symfony/recipes/pull/1276